src/seedu/addressbook/Main.java

```diff
@@ -107,6 +107,11 @@ private CommandResult executeCommand(Command command)  {
         try {
             command.setData(addressBook, lastShownList);
             CommandResult result = command.execute();
+            
+        if(storage.isFileDeleted()) {
+           	ui.showToUser(storage.getWarningMessage());
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.